CHANGELOG.md

```diff
@@ -1,6 +1,139 @@
 Change Log
 ==========
 
+# Version 3.6.0
+
+_2022-09-01_
+
+This version brings initial support for `@defer` as well as data builders.
+
+## 💙️ External contributors
+
+Many thanks to @engdorm, @Goooler, @pt2121 and @StylianosGakis for their contributions!
+
+## ✨️ [new] `@defer` support
+
+`@defer` is [stage 1 GraphQL spec proposal](https://github.com/graphql/graphql-spec/blob/main/CONTRIBUTING.md#stage-1-proposal) to allow incremental delivery of response payloads. 
+
+`@defer` allows you to mark a fragment as deferrable, meaning it can be omitted in the initial response and delivered as a subsequent payload. This improves latency for all fields that are not in that fragment. You can read more about `@defer` in the [RFC](https://github.com/graphql/graphql-wg/blob/main/rfcs/DeferStream.md) and contribute/ask question in the [`@defer` working group](https://github.com/robrichard/defer-stream-wg).
+
+Apollo Kotlin supports `@defer` by default and will deliver the successive payloads as `Flow` items. Given the below query:
+
+```graphql
+query GetComputer {
+  computer {
+    __typename
+    id
+    ...ComputerFields @defer
+  }
+}
+
+fragment ComputerFields on Computer {
+  cpu
+  year
+  screen {
+    resolution
+  }
+}
+```
+
+And the following server payloads:
+
+**payload 1**:
+```json
+{
+  "data": {
+    "computer": {
+      "__typename": "Computer",
+      "id": "Computer1"
+    }
+  },
+  "hasNext": true
+}
+```
+
+**payload 2**:
+```json
+{
+  "incremental": [
+    {
+      "data": {
+        "cpu": "386",
+        "year": 1993,
+        "screen": {
+          "resolution": "640x480"
+        }
+      },
+      "path": [
+        "computer",
+      ]
+    }
+  ],
+  "hasNext": true
+}
+```
+
+You can listen to payloads by using `toFlow()`:
+
+```kotlin
+apolloClient.query(query).toFlow().collectIndexed { index, response ->
+  // This will be called twice
+
+  if (index == 0) {
+    // First time without the fragment
+    assertNull(response.data?.computer?.computerFields)
+  } else if (index == 1) {
+    // Second time with the fragment
+    assertNotNull(response.data?.computer?.computerFields)
+  }
+}
+```
+
+You can read more about it in [the documentation](https://www.apollographql.com/docs/kotlin/fetching/defer).
```

Comment:
Will this link to docs be live when we release this changelog?

Reply:
Aha I see the file below. 😎

Reply:
Yep, it's a bit of a transition state right now but once merged, it should all work